integration-tests/src/test/java/com/transferwise/tasks/triggering/KafkaTasksExecutionTriggererIntTest.java

```diff
@@ -0,0 +1,126 @@
+package com.transferwise.tasks.triggering;
+
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.transferwise.common.baseutils.UuidUtils;
+import com.transferwise.tasks.BaseIntTest;
+import com.transferwise.tasks.ITaskDataSerializer;
+import com.transferwise.tasks.ITasksService;
+import com.transferwise.tasks.ITasksService.AddTaskRequest;
+import com.transferwise.tasks.TasksProperties;
+import com.transferwise.tasks.domain.BaseTask;
+import com.transferwise.tasks.handler.SimpleTaskProcessingPolicy;
+import com.transferwise.tasks.helpers.kafka.partitionkey.IPartitionKeyStrategy;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import lombok.SneakyThrows;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.assertj.core.api.Assertions;
```

Comment:
AssertJ turned to out to make compilation very slow. We want to move away from it in libraries.
Please refactor to use Jupiter assertions.

Reply:
Done